### ISSUE TITLE
nexrender ftp upload fails on windows

### ISSUE THREAD

**Issue by benjioh5** [inlife/nexrender#555]
**Describe the bug**
```
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: Please login with USER and PASS.
    at makeError (D:\nexrender\node_modules\ftp\lib\connection.js:1067:13)
    at Parser.<anonymous> (D:\nexrender\node_modules\ftp\lib\connection.js:113:25)
    at Parser.emit (events.js:315:20)
    at Parser._write (D:\nexrender\node_modules\ftp\lib\parser.js:59:10)
    at writeOrBuffer (internal/streams/writable.js:358:12)
    at Parser.Writable.write (internal/streams/writable.js:303:10)
    at Socket.ondata (D:\nexrender\node_modules\ftp\lib\connection.js:273:20)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
Emitted 'error' event on  instance at:
    at Object.reentry [as cb] (D:\nexrender\node_modules\ftp\lib\connection.js:192:14)
    at Parser.<anonymous> (D:\nexrender\node_modules\ftp\lib\connection.js:113:22)
    at Parser.emit (events.js:315:20)
    [... lines matching original stack trace ...]
    at readableAddChunk (internal/streams/readable.js:284:9) {
  code: 530
}
```

**Information about environment**
* Which version of nexrender are you using?
* What kind of the setup do you have? (server+worker or cli)
* Are you using pre-built binaries or globally installed npm packages?
* Which kind (and version) of operating system you are running it in?
* Any additional information that might be helpful

**Expected behavior**

**Screenshots**

**Comment by benjioh5** [comment]
I Think this is associated with this issue https://github.com/mscdex/node-ftp/issues/270

**Comment by inlife** [comment]
Added a commit that should fix that: 712697b
Thanks for finding out the related issue @benjioh5

**Comment by benjioh5** [comment]
Thx!
